src/microbe_stage/FloatingChunk.cs

```diff
@@ -225,6 +229,11 @@ public override void _Ready()
         InitPhysics();
     }
 
+    public override string ToString()
+    {
+        return debugName;
+    }
```

Comment:
Perhaps we can just directly return the formatted debug name here?

Reply:
that's true.

Reply:
chunktype.Name is not stored inside FloatingChunk class, so I can't get the name if elsewhere.

Reply:
Add a new Name property to FloatingChunk and assign it with `chunkType.Name` in the Init. Not sure why it's missing in the first place...

Reply:
Maybe because Node has a Name property

Reply:
I guess there's never been a need for chunks to store that information so I never added such a field. One thing I wonder though is that there's that method that makes a chunk configuration from an existing chunk somewhere and I think that copies over the name somehow.

Reply:
Name is not included.

Reply:
> Maybe because Node has a Name property

Ah yeah that's true, then this should have more explicit naming to `ChunkName`.

> Name is not included.

Then there is high possibility that the Name is not copied over in that method.